src/app/components/input.component.ts

```diff
@@ -140,7 +140,10 @@ export class OebInputComponent {
 
 	ngAfterViewInit() {
 		if (this.autofocus) {
-			this.focus();
+			// delay focus to prevent unwanted scrolling
```

Comment:
Is this really still necessary considering the other change you made?

Reply:
For the input fields we have now it is not necessary, but i dont think it hurts to add it, so that we dont run into the same thing in the future?